espn_api/baseball/team.py

```diff
@@ -5,7 +5,7 @@
 
 class Team(object):
     '''Teams are part of the league'''
-    def __init__(self, data, roster, schedule, year, **kwargs):
+    def __init__(self, data, roster, schedule, year, owners, **kwargs):
         self.team_id = data['id']
```

Comment:
I think for each team class all you will need to add is `self.owners = data.get('owners', [])`